SCF-Session-8/zooManagement/Cage.java

```diff
@@ -0,0 +1,67 @@
+package zooManagement;
+
+import java.util.ArrayList;
+
+/**
+ * The Cage class will add the cage to the Zoo
+ * @author Ishita_Mathur
+ * Here many getter setter are present to initializes the Cage properties
+ */
+public class Cage
+{
+	private AnimalName animalType;
+	private int capacity;
+	ArrayList<Animal> listOfAnimals = new ArrayList<>();
```

Comment:
If you are declaring the ArrayList of type <animal> then it should be instantiated with same type.